ingress/controllers/nginx/README.md

```diff
@@ -149,6 +150,23 @@ By default the controller redirects (301) to HTTPS if there is a TLS Ingress rul
 To disable this behavior use `hsts=false` in the NGINX config map.
 
 
+### Automated Certificate Management with Kube-Lego
+
+[Kube-Lego] automatically requests missing certificates or expired from
+[Let's Encrypt] by monitoring ingress resources and its referenced secrets. To
+enable this for an ingress resource you have to add an annotation:
+
+```
+kubectl annotate ing ingress-demo kubernetes.io/tls-acme="true"
```

Comment:
what are the requirements of an ingress controller (if any) for this annotation to work? please inline, eg: will it work with cloudprovider loadbalancers too?